test/functional/ex_cmds/encoding_spec.lua

```diff
@@ -25,7 +25,7 @@ describe('&encoding', function()
     clear('--cmd', 'set enc=latin1')
```

Comment:
e_afterinit should not be used, as there is no difference in behavior for and after vimenter.

and 'termguicolor' should be fixed instead. Add `ui_call_gui_option_changed` and include 'termguicolor' amoung the gui options.